docs/guides/multicloud_cluster_with_baremetal_servers.md

```diff
@@ -26,7 +26,7 @@ as a node by the apiserver. This can be achieved manually ([method A](#method-a-
 resource "scaleway_k8s_cluster" "multicloud" {
   name    = "multicloud-cluster"
   type    = "multicloud"
-  version = "1.27.1"
```

Comment:
We would change this all the time, why not using a data source with latest to have an example that is always up to date and can be copy and pasted easily?

Reply:
There is multiple snippets that should work with a simple copy/paste, using a valid version help that.
We could use a minor version `X.Y` but it would require enabling auto-upgrade